Tests/AppStoreConnectClientTests/AppStoreConnectClientTests.swift

```diff
@@ -168,4 +168,155 @@ final class AppStoreConnectClientTests: XCTestCase {
             XCTFail("Unexpected error: \(error)")
         }
     }
+    
+    func testFetchBuildsSuccess() async throws {
+        var mockClient = MockAPIClient()
+        mockClient.result = .ok
+        let client = AppStoreConnectClient(client: mockClient)
+        let app = Application(id: "Foo", bundleId: "Bar")
+        let expectedBuilds = [Build(schema: MockObjects.schema)]
+        let build = try await client.fetchBuilds(for: app)
+        XCTAssertEqual(build.first?.id, expectedBuilds.first?.id)
+        XCTAssertEqual(build.first?.version, expectedBuilds.first?.version)
+        XCTAssertEqual(build.first?.uploadedDate, expectedBuilds.first?.uploadedDate)
+        XCTAssertEqual(build.first?.minOsVersion, expectedBuilds.first?.minOsVersion)
+    }
+    
+    func testFetchBuildsBadRequest() async throws {
+        var mockClient = MockAPIClient()
+        mockClient.result = .badRequest
+        let client = AppStoreConnectClient(client: mockClient)
+        let app = Application(id: "Foo", bundleId: "Bar")
+        do {
+            _ = try await client.fetchBuilds(for: app)
+            XCTFail("Expected error not thrown")
+        } catch AppStoreConnectError.badRequest {
```

Comment:
Why you do not check and handle error.
Add check to catch.

Reply:
Test added to catch.